src/map/entities/battleentity.cpp

```diff
@@ -2064,22 +2064,23 @@ bool CBattleEntity::OnAttack(CAttackState& state, action_t& action)
         else if ((xirand::GetRandomNumber(100) < attack.GetHitRate() || attackRound.GetSATAOccured()) &&
                  !PTarget->StatusEffectContainer->HasStatusEffect(EFFECT_ALL_MISS))
         {
-            // attack hit, try to be absorbed by shadow unless it is a SATA attack round
-            if (!(attackRound.GetSATAOccured()) && battleutils::IsAbsorbByShadow(PTarget, this))
-            {
-                actionTarget.messageID = MSGBASIC_SHADOW_ABSORB;
-                actionTarget.param     = 1;
-                actionTarget.reaction  = REACTION::EVADE;
-                attack.SetEvaded(true);
-            }
-            else if (attack.IsParried())
+            // attack hit, try to be deflected by parry unless it is a SATA attack round
+            if (!(attackRound.GetSATAOccured()) && attack.IsParried())
```

Comment:
I know this is a corner case, and I know this is probably true, but do you have a source for not being able to parry a TA? Just curious (this isn't my repo)

Reply:
It's in there more for the execution of a Hide > Sneak Attack from the front combo, which shouldn't be capable of being missed/parried AFAIK. I would assume that's also true for Trick Attack as a result.